modules/op-setting-up-openshift-pipelines-to-view-project-vulnerabilities.adoc

```diff
@@ -0,0 +1,137 @@
+// This module is included in the following assemblies:
+// * secure/setting-up-openshift-pipelines-to-view-software-supply-chain-security-elements.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="op-setting-up-openshift-pipelines-to-view-project-vulnerabilities_{context}"]
+= Setting up Openshift pipelines to view project vulnerabilities
+
+The PipelineRun details view provides a visual representation of identified vulnerabilities,  categorized by the severity (critical, high, medium, and low). This streamlined view facilitates prioritization and remediation efforts.
+
+.Viewing vulnerabilities on the PipelineRun details view
+image::vulnerabilities_details.png[]
+
+You can also review the vulnerabilities in the Vulnerabilities column in the PipelineRuns list view page.
```

Comment:
I would suggest "pipeline run list view". "PipelineRun" when spelled like this is a CR name and the plural of this doesn't really work, though it is common in developer jargon.